### ISSUE TITLE
Unmatched quote character causes code to be colored as a string

### ISSUE THREAD

**Issue by Yethal** [nushell/vscode-nushell-lang#40]
Paste following line into VSCode with the extension installed
````
vboxmanage list vms|lines|str trim -c '"' -a|split column " " name uid
````
Notice everything after ````'"'```` (double quote enclosed in single quotes) is highlighted in green as if it were a string

**Comment by fdncred** [comment]
Just noticed this. sorry. I'm open to fixes. I'm not very good at typescript.

**Comment by fdncred** [comment]
should be fixed with #73
